registry/client.go

```diff
@@ -62,6 +65,30 @@ type credentials struct {
 	password string
 }
 
+type schemaInfoPayload struct {
+	Schema  string `json:"schema"`
+	ID      int    `json:"id"`
+	Version int    `json:"version"`
+}
+
+// Parse converts the string schema registry response into a
+// SchemaInfo object with an avro.Schema schema
+func (s *schemaInfoPayload) parse() (info SchemaInfo, err error) {
```

Comment:
There is no need to privatise this function. IMO it gives the impression it is used internally by `schemaInfoPayload` and is not meant for use by other functions. The struct is already private.

Reply:
ok that makes senes. fixed!